change/@microsoft-teams-js-4fe5ca49-b46c-4a6b-b580-bd6035df67e6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Keep the common data models in the `externalAppCardAction` API file to share with the `externalAppCardActionForCEA` API.",
```

Comment:
>Keep the common data models in the `externalAppCardAction` API file to share with the `externalAppCardActionForCEA` API. [](http://example.com/codeflow?start=14&length=120)

Consider:
"Unified common data models for external card actions into `externalAppCardActions` namespace"

Reply:
Updated with your suggestion. Thanks!